privilege/privileges/privileges.go

```diff
@@ -158,7 +158,7 @@ func (p *UserPrivileges) ConnectionVerification(user, host string, authenticatio
 	}
 
 	p.user = user
```

Comment:
Should it be `p.user = u` here?

I'm not sure could the privilege record be something like 
user = '%' and host = '192.168.%'

Reply:
@tiancaiamao MySQL dosen't support wildcard in user name.